docker/Dockerfile

```diff
@@ -14,7 +14,7 @@ USER appuser
 WORKDIR /home/appuser
 
 ENV PATH="/home/appuser/.local/bin:${PATH}"
-RUN wget https://bootstrap.pypa.io/pip/3.6/get-pip.py && \
```

Comment:
This change is incorrect. The python in this docker container is 3.6.

Reply:
Thanks Yuxin for comment! So we should keep this Dockerfile unchanged? What about the `deploy.Dockerfile` file

Reply:
Best option: make `timm/fairscale` optional so that D2 still works with python 3.6; keep both dockerfile unchanged.

fairscale actually requires python 3.8 now: https://github.com/facebookresearch/fairscale/commit/ca18b50cbe0a48ad7a0ca3f1962e5f88f79d717b

Reply:
Let's land this PR to fix the CI first and then make fairscale optional make it more flexible